requirements.txt

```diff
@@ -1,16 +1,31 @@
-psutil==2.2.0
-requests>=2.7.13
-netifaces==0.10.4
-kafka-python==1.3.1
-pykafka==1.1.0
-kafka==1.3.3
-docker-py==1.10.6
-python-dateutil==2.4.2
-semantic_version==2.5.0
-Yapsy==1.11.223
+backports.ssl-match-hostname==3.7.0.1
```

Comment:
Plugin-specific dependencies should not be added to the global list, but installed inside the plugin itself.
Think about the case where one isn't using the emitter but yet ends up installing all the unnecessary packages.

Reply:
@sahilsuneja1 Okay I made the change suggested and rolled back the global list of packages in `requirements.txt` file.